cmd/mattermost/commands/team.go

```diff
@@ -247,3 +257,29 @@ func listTeamsCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func searchTeamCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	teams := make([]*model.Team, 0, len(args))
```

Comment:
The size of the teams doesn't depends on the len of the args, I would use here simply a `var teams []*model.Team` (You don't need to initialize it, because the default value would be an empty list).